web/src/lib/components/shared-components/create-share-link-modal/create-shared-link-modal.svelte

```diff
@@ -41,27 +41,22 @@
 
   $: shareType = albumId ? SharedLinkType.Album : SharedLinkType.Individual;
 
-  onMount(async () => {
```

Comment:
Hello, why do we remove `onMount` here?

Reply:
Because `check:svelte` complained about it? I'm just getting into svelte, so perhaps it needs to remain? No longer needing to make this call within

```
const module = await import('copy-image-clipboard');
```